optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/exceptions/OptashiftExceptionMapper.java

```diff
@@ -0,0 +1,24 @@
+package org.optaplanner.openshift.employeerostering.server.exceptions;
```

Comment:
I'd put this in the package "common", in the philosophy that we have these packages under each module package:
- common (code used by or extended all other packages)
- skill (normal package for functionalty)
- employee (normal package for functionalty)
- spot (normal package for functionalty)
- ... (normal packages based on functionalty, NOT type).
- app (bootstrap code and other code that use all other packages)

I'll merge this PR and do that adjustment myself.